api/v1/monitoring_types.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MonitoringSpec defines the desired state of Monitoring
+type MonitoringSpec struct {
+}
+
+// MonitoringStatus defines the observed state of Monitoring
+type MonitoringStatus struct {
+	// State provides user-readable status.
+	State string `json:"state,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
```

Comment:
The following annotation appears to be missing:
`// +kubebuilder:resource:scope=Cluster`. Otherwise Monitoring would become a namespaced resource.